app/views/govuk_publishing_components/audit/show.html.erb

```diff
@@ -2,284 +2,291 @@
 
 <%= render 'govuk_publishing_components/components/title', title: "Components audit", margin_top: 0; %>
 
-<% if ENV["MAIN_COMPONENT_GUIDE"] %>
-  <div class="govuk-tabs" data-module="govuk-tabs">
-    <h2 class="govuk-tabs__title">
-      Contents
-    </h2>
-    <ul class="govuk-tabs__list">
-      <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
-        <a class="govuk-tabs__tab" href="#applications">
-          Applications
-        </a>
-      </li>
-      <li class="govuk-tabs__list-item">
-        <a class="govuk-tabs__tab" href="#components-gem">
-          Components
-        </a>
-      </li>
-    </ul>
-    <div class="govuk-tabs__panel" id="applications">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: "Applications",
-        font_size: "l",
-        margin_bottom: 6
-      } %>
+<div class="govuk-tabs" data-module="govuk-tabs">
+  <h2 class="govuk-tabs__title">
+    Contents
+  </h2>
+  <ul class="govuk-tabs__list">
+    <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+      <a class="govuk-tabs__tab" href="#applications">
+        Applications
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#components-gem">
+        Components
+      </a>
+    </li>
+  </ul>
+  <div class="govuk-tabs__panel" id="applications">
+    <%
+      title = "Applications"
+      title = "This application" unless ENV["MAIN_COMPONENT_GUIDE"]
+    %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: title,
+      font_size: "l",
+      margin_bottom: 6
+    } %>
 
-      <% if @applications.any? %>
-        <details class="govuk-details" data-module="govuk-details">
-          <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
-              How to use this information
-            </span>
-          </summary>
-          <div class="govuk-details__text">
-            <p class="govuk-body">This page shows information about component use on GOV.UK. This information has been cross referenced with the components in the gem to produce warnings where e.g. a print stylesheet for a component exists but has not been included in an application.</p>
-            <p class="govuk-body">Warnings should be investigated, although there may be a reason why the application has been configured as it is. Note that 'code' can refer to templates or ruby code.</p>
-          </div>
-        </details>
+    <% if @applications.any? %>
+      <details class="govuk-details" data-module="govuk-details">
```

Comment:
Similarly here, any reason for not using the component?

Reply:
Fixed, thanks.